js/scripts.js

```
@@ -33,7 +33,9 @@ const checkRadioByValue = function (radio_array, value) {
     return;
   }
   value = value.toString();
-  radio_array.find(radio => radio.value == value).checked = true;
+
+  if (typeof radio_array.find(radio => radio.value == value) !== 'undefined')
+    radio_array.find(radio => radio.value == value).checked = true;
```

This change does not fix the problem, but just masks it by swallowing the error. The actual issue is caused by an invalid value being passed to `fillFields()` on line 86 of the file.

```js
fillFields([], false, 'unknown')
```
should actually be
```js
fillFields([], false, -1)
```